popsycle/run.py

```diff
@@ -103,7 +103,14 @@ def run():
     optional.add_argument('--overwrite',
                           help="Overwrite all output files.",
                           action='store_true')
-    args = parser.parse_args()
+
+
+    optional.add_argument('--pbh-config-filename', type=str,
+                      help='Name of configuration file containing '
+                           'pbh inputs. Default if needed is: '
+                           'pbh_config.yaml',
+                      default=None)
```

Comment:
This needs to have a default value of 'pbh_config.yaml', as described in the help string.